internal/evaluator/evaluator.go

```diff
@@ -306,3 +265,20 @@ func TransformAdditionalInstances(
 
 	return additionalInstances, nil
 }
+
+func convertFS(apiFS *api.FileSystem) (fs fs.FileSystem, err error) {
+	fs = failing.New(ErrNoFS)
```

Comment:
Should it be populated with `request.FilesContents` for backward compatibility?

Reply:
It seems that we only use it tests, so turns out no backwards compatibility is needed.